src/drive/web/modules/filelist/File.jsx

```diff
@@ -351,14 +351,15 @@ class File extends Component {
             }}
           />
         )}
-        {actionMenuVisible && (
-          <ActionMenu
-            file={attributes}
-            reference={this.filerowMenuToggle}
-            actions={actions}
-            onClose={this.hideActionMenu}
-          />
-        )}
+        {actions &&
```

Comment:
We can have the ActionMenu displayed when we don't have any action? Or is it because sometimes this component is rendered before we have data ?

Reply:
There are views where files don't have any actions (moving view, interapp folder picking…), and in that case we don't want to show the button. It's the same for the [`ActionMenu`](https://github.com/cozy/cozy-drive/blob/b17e10a181c744eb7166018842b131853ae5cbcd/src/drive/web/modules/filelist/File.jsx#L354).